package.json

```diff
@@ -29,51 +29,53 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
+    "@floating-ui/dom": "^1.2.5",
     "babel-plugin-filter-imports": "^4.0.0",
     "babel6-plugin-strip-class-callcheck": "^6.0.0",
     "broccoli-funnel": "^3.0.5",
+    "ember-auto-import": "^2.6.1",
     "ember-classic-decorator": "^2.0.0",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1",
     "ember-cli-sass": "^10.0.1",
     "ember-decorators": "^6.1.1",
-    "ember-popper": "^0.11.3",
     "ember-truth-helpers": "^3.0.0",
     "sass": "^1.35.2"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "~2.2.8",
+    "@ember/test-waiters": "^3.0.2",
     "@html-next/flexi-default-styles": "^2.2.0",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
+    "ember-cli": "~3.26.1",
```

Comment:
no changes in the `ember-cli` version, just sorting the devDependencies alphabetically